web/src/OverviewGrid.tsx

```diff
@@ -16,6 +16,12 @@ let OverviewGridRoot = styled.ul`
   padding-left: ${SizeUnit(0.25)};
   position: relative;
   box-sizing: border-box;
+  &:nth-child(even) {
```

Comment:
i think you can remove the nth-child(even) selector?

the grid is always preceded by a divider, so this is currently a no-op (and makes the code more brittle)

Reply:
Ah yes thanks @nicks . Fixing.